parcel/ParcelModel.jl

```diff
@@ -141,6 +141,7 @@ function parcel_model(dY, Y, p, t)
     dqᵥ_dt = -dqₗ_dt_v2l - dqᵢ_dt_v2i
 
     dSₗ_dt =
+    # prescribed_thermodynamics ? AIDA_rate(t, t_profile, T_profile) / (-0.0065) :
```

Comment:
Is that line supposed to be commented out?

Reply:
Yes, it wasn't working right (saturation was order of magnitude 1000) so I'm going to use an approximate of the updraft for now

Reply:
Then we should figure out why. We can't run calibrations with the updraft version because our pessure is 1000x ff then?